### ISSUE TITLE
Dev branch broken after merge

### ISSUE THREAD

**Issue by stevebalk** [memaar92/transcendence#4]
The backend will restart over and over with the same error message:

```
Traceback (most recent call last):
  File "/usr/local/bin/daphne", line 8, in <module>
    sys.exit(CommandLineInterface.entrypoint())
  File "/usr/local/lib/python3.10/site-packages/daphne/cli.py", line 171, in entrypoint
    cls().run(sys.argv[1:])
  File "/usr/local/lib/python3.10/site-packages/daphne/cli.py", line 233, in run
    application = import_by_path(args.application)
  File "/usr/local/lib/python3.10/site-packages/daphne/utils.py", line 17, in import_by_path
    target = importlib.import_module(module_path)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/app/backend/./backend/asgi.py", line 20, in <module>
    "http": get_asgi_application(),
  File "/usr/local/lib/python3.10/site-packages/django/core/asgi.py", line 12, in get_asgi_application
    django.setup(set_prefix=False)
  File "/usr/local/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.10/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.10/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/app/backend/./usermanagement/models.py", line 3, in <module>
    from .utils import random_filename
ModuleNotFoundError: No module named 'usermanagement.utils'
```

Seems like it could not find usermanagment.utils

**Comment by stevebalk** [comment]
https://github.com/memaar92/transcendence/blob/75b17fc8892acb1841de5aead8f81922c3cba704/backend/usermanagement/models.py#L3C1-L4C1

Hier wird ein utils file imported was es nicht gibt

**Comment by stevebalk** [comment]
Fixed with #3

**Comment by stevebalk** [comment]
Fixed with #3

**Comment by stevebalk** [comment]
Fixed with https://github.com/memaar92/transcendence/pull/3
